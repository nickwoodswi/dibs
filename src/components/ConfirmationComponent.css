.confirmation-component{
    font-style: italic
}
.hold-order{
    color: red
}