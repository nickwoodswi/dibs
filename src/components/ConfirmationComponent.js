import React, {Component} from 'react'
import './ConfirmationComponent.css'

class ConfirmationComponent extends Component {
    render() {
        return(
            <span className="confirmation-component"><h4><span className="hold-order">{this.props.holdOrder}H</span> - {this.props.holdDate}</h4></span>
        )
    }
}

export default ConfirmationComponent