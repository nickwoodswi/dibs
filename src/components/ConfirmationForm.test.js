import React from 'react';
import { render } from '@testing-library/react';
import ConfirmationComponent from './ConfirmationComponent';
import { MemoryRouter } from 'react-router-dom'
import renderer from 'react-test-renderer'

test('renders confirmation form', () => {
    render(<MemoryRouter><ConfirmationComponent /></MemoryRouter>);
});

it('ConfirmationForm renders the UI as expected', () => {
    const tree = renderer
        .create(<MemoryRouter><ConfirmationComponent /></MemoryRouter>)
        .toJSON();
    expect(tree).toMatchSnapshot();
});