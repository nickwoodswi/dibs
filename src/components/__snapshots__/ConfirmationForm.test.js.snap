// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`ConfirmationForm renders the UI as expected 1`] = `
<span
  className="confirmation-component"
>
  <h4>
    <span
      className="hold-order"
    >
      H
    </span>
     - 
  </h4>
</span>
`;
